src/views/Article/Index.jsx

```diff
@@ -33,28 +55,77 @@ const Article = (props) => {
       <Container>
         <ArticleTitle title={article.title} />
         <ArticleDescription description={article.description} />
-        <ImageContainer src={article.images[0] || 'https://res.cloudinary.com/djdsxql5q/image/upload/v1554806589/Authors%20Haven/culture.jpg'} />
+        <ImageContainer
+          src={
```

Comment:
@leksyib I mentioned this before about saving long URLs or ternaries like this to a variable to make the code look cleaner